lib/src/asset/asset.dart

```diff
@@ -13,5 +13,10 @@ class Asset {
 
   Asset(this.id, this.stringContents);
 
+  bool operator ==(other) =>
```

Comment:
I am actually just going to make this a test only utility, thats all I was using it for.